src/controllers/resetPassword.js

```diff
@@ -0,0 +1,49 @@
+import bcrypt from 'bcrypt'
+import dotenv from 'dotenv'
+import { Employee } from '../../models/'
+
+dotenv.config()
+
+// Reset password controller function
+const resetPassword = ('/', async (req, res) => {
```

Comment:
find a way to put this / in the router if possible. The '/'.

Reply:
The changes requested have been addressed, please review and approve!